src/store/activity/actions.js

```diff
@@ -22,7 +22,7 @@ import { formatCirclesValue } from '~/utils/format';
 
 const { ActivityTypes, ActivityFilterTypes } = core.activity;
 
-export const PAGE_SIZE = 10;
+export const PAGE_SIZE = 2;
```

Comment:
Was this changed for testing purposes or is this an intentional change?

Reply:
is this same constant also used in the request to the core? That is important

Reply:
It should be 10, I corrected it. `PAGE_SIZE` is send to core